docs/sources/datasources/loki.md

```diff
@@ -20,13 +20,13 @@ You can run Loki on your own hardware or use [Grafana Cloud](https://grafana.com
 
 To access Loki settings, click the **Configuration** (gear) icon, then click **Data Sources**, and then click the Loki data source.
 
-| Name                  | Description                                                                                                                                               |
-| --------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `Name`                | The data source name. This is how you refer to the data source in panels, queries, and Explore.                                                           |
-| `Default`             | Default data source that is pre-selected for new panels.                                                                                                  |
-| `URL`                 | URL of the Loki instance, e.g., `http://localhost:3100`.                                                                                                  |
-| `Whitelisted Cookies` | Grafana Proxy deletes forwarded cookies by default. Specify cookies by name that should be forwarded to the data source.                                  |
-| `Maximum lines`       | Upper limit for the number of log lines returned by Loki (default is 1000). Lower this limit if your browser is sluggish when displaying logs in Explore. |
+| Name               | Description                                                                                                                                               |
```

Comment:
```suggestion
| Names               | Descriptions                                                                                                                                               |
```

Reply:
i'm not sure this should be plural?

Reply:
By default, use plural headings for consistency. We on the Docs Squad get to argue about such details. :)

Reply:
The disadvantage of not having consistency, is that some headers will read "Name" and others will read "Names" and depending on the team who owns the technical content across entire product lines.

Reply:
An advantage of using plural is that the minute you have more than one name and description, which is an ingredient for a list in the first place, the titles are already plural.

Reply:
> By default, use plural headings for consistency. We on the Docs Squad get to argue about such details. :)

We have "argued" and come to consensus at https://github.com/grafana/technical-documentation/issues/76 to use the singular form. :)